app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -194,3 +194,8 @@ span.btn a{ text-decoration: none; color: #333333}
 body.modal-open {
   overflow: visible;
 }
+
+
+.glyphicon-copy:before {
+  content: "\f121";
```

Comment:
whats the background here? if its a workaround, from which version should we remove it?